test/extended/topology_manager/README.md

```diff
@@ -0,0 +1,32 @@
+# Topology Manager testsuite
+
+## summary
+
+Tests in this directory tree check the behaviour of the topology manager.
+To work correctly, the topology manager orchestrates various other components like cpu manager and device manager.
+Furthermore, the topology manager is most useful to align device resources, like SRIOV cards.
```

Comment:
Maybe change to SR-IOV VFs instead of cards.

Reply:
done